### PR TITLE
[stable/prometehus-blackbox-exporter] Improve configurability.

### PR DESCRIPTION
**Pull request by ghost**
No

#### What this PR does / why we need it:

[`stable/prometehus-blackbox-exporter`] Improve configurability:
* `values.yaml`:
  * Add: `priorityClassName`, `podDisruptionBudget`, `strategy`.
* `templates/deployment.yaml`:
  * Parameterize: `strategy`.
  * Add: `priorityClassName`.
* `templates/poddisruptionbudget.yaml`:
  * Add: new file.

#### Which issue this PR fixes

None

#### Special notes for your reviewer:

None

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by desaintmartin** [comment]
Thanks!
/ok-to-test

**Comment by josdotso** [comment]
>  tide Pending — Not mergeable. Needs approved, lgtm labels. 

Can you help with this @desaintmartin ?  thanks

File: stable/prometheus-blackbox-exporter/templates/poddisruptionbudget.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "prometheus-blackbox-exporter.fullname" . }}
+  labels:
+    app: {{ template "prometheus-blackbox-exporter.name" . }}
```
Comment (desaintmartin) [inline stable/prometheus-blackbox-exporter/templates/poddisruptionbudget.yaml]: Could you use the new labels?

File: stable/prometheus-blackbox-exporter/templates/poddisruptionbudget.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    chart: {{ template "prometheus-blackbox-exporter.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "prometheus-blackbox-exporter.name" . }}
```
Comment (desaintmartin) [inline stable/prometheus-blackbox-exporter/templates/poddisruptionbudget.yaml]: Same

**Review by desaintmartin — CHANGES_REQUESTED** [review]

File: stable/prometheus-blackbox-exporter/templates/poddisruptionbudget.yaml (hunk 1 above)
Comment (ghost) [reply stable/prometheus-blackbox-exporter/templates/poddisruptionbudget.yaml]: I believe this is fixed in latest commit.

File: stable/prometheus-blackbox-exporter/templates/poddisruptionbudget.yaml (hunk 2 above)
Comment (ghost) [reply stable/prometheus-blackbox-exporter/templates/poddisruptionbudget.yaml]: I believe this is fixed in latest commit.

**Comment by josdotso** [comment]
Sorry for the noise. Took a moment to get the sign-off right.

**Comment by desaintmartin** [comment]
We miss a way to disable the PDB if not needed, let me do a PR.

**Pull request merged**